src/tools/build-manifest/src/main.rs

```diff
@@ -81,6 +81,7 @@ static TARGETS: &'static [&'static str] = &[
     "s390x-unknown-linux-gnu",
     "sparc64-unknown-linux-gnu",
     "wasm32-unknown-emscripten",
+    "wasm32-unknown-unknown",
```

Comment:
This line is not necessary.

This list is defining the targets we distribute as part of Rust releases, and this target will not be built and distributed at this time.